lib/index.d.ts

```diff
@@ -452,6 +453,21 @@ declare class Realm {
      * @returns void
      */
     write(callback: () => void): void;
+
+    /**
+     * @returns void
```

Comment:
We should add the comments from the documentation .js file here as well -- hopefully at some point, this file can be the single source of documentation both for the website and for typescript-aware ide's